assignment3.txt

```diff
@@ -0,0 +1,10 @@
+Ava N.
+June 30, 2016
+
+Assignment 3
+
+1. The code written was invalid because the printf statement should look like this: printf("%d\n",i);
+2. i=(int) should be before the for loop. Also, the statement should look like this: int i;
+3. i.2^n
```

Comment:
O(n^2) actually! The for loops are nested so you'd multiply their time complexities, so n*n = n^2